README.md

```diff
@@ -18,15 +18,26 @@ These are some of the required steps, tips, and tricks when it comes to running
 
 ### Prerequisites
 
-1. Newest [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+1. .NET SDK matching your service (the latest App Template uses [.NET 8 SDK](https://dotnet.microsoft.com/download/dotnet/8.0), older versions may also require [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0))
 2. Newest [Git](https://git-scm.com/downloads)
 3. A code editor - we like [Visual Studio Code](https://code.visualstudio.com/Download)
     - Also
       install [recommended extensions](https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions) (
       f.ex. [C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp))
-4. [Podman Desktop](https://podman-desktop.io)/[Podman](https://podman.io) (Linux users can also use native Docker) or [Docker Desktop](https://www.docker.com/products/docker-desktop) (Windows and Mac) This might require you to purchase a license.
-On mac with apple silicone (M1, M2, M3):
-5. [vfkit](https://github.com/crc-org/vfkit?tab=readme-ov-file#installation)
+4. Tooling to run containers:  
+    - [Podman](https://podman.io)* (optionally with [Podman Desktop](https://podman-desktop.io))
+    - [Docker Desktop](https://www.docker.com/products/docker-desktop) (Windows and Mac) This might require you to purchase a license. Linux users can also use native Docker.
```

Comment:
> Linux users can also use native Docker.

Maybe this part could be a separate bullet point? WSL Linux with native Docker install using systemd is probably the easiest way (it was for me atleast)